job_templates/sag_pt/config_fed_client.conf

```diff
@@ -55,10 +55,10 @@
     }
   ],
 
-  # this defined an array of task data filters. If provided, it will control the data to server
+  # this defined an array of task data filters. If provided, it will control the data from server controller to client executor
   task_data_filters =  []
 
-  # this defined an array of task result filters. If provided, it will control the result to server
+    # this defined an array of task result filters. If provided, it will control the result from client executor to server controller
   task_result_filters = []
```

Comment:
To add more to this, we also have client side controller,
What is a more accurate way to describe this?
Should we explain the "direction" here (IN/OUT/INOUT)?
Or we should not expose too many details?

What do you think @yanchengnv @yhwen ?